scripts/chemistry/periodic_table/process.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Generate nodes for all elements in the periodic table.
+"""
+
+import numpy as np
+import pandas as pd
+
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_list('input_csv', 'periodic_table.csv,compounds.csv',
+                  'CSV file with properties of all elements.')
+flags.DEFINE_string('output_prefix', 'substance',
+                    'Output path for csv and tmcf files.')
+
+_OUTPUT_COLUMNS = [
+    'Name',
+    'TypeOf',
+    'ChemicalName',
+    'Symbol',
+    'Type',
+    'AtomicNumber',
+]
+
+_NODE_TEMPLATE = """
+Node: E:Substance->E0
+typeOf: C:Substance->TypeOf
+dcid: C:Substance->Name
+name: C:Substance->Name
+chemicalName: C:Substance->ChemicalName
+chemicalSymbol: C:Substance->Symbol
+atomicNumber: C:Substance->AtomicNumber
+chemicalSubstanceType: C:Substance->Type
+"""
+
+
+def _process_file(input_csv: str) -> pd.DataFrame:
+    """Read the data from the input csv, clean it up and return a DataFrame
+
+  Args:
+    input_csv: CSV downloaded from the data source
+
+  Returns:
+    DataFrame with the output columns from the input
+  """
+
+    df = pd.read_csv(input_csv)
+
+    # Remove extra whitespaces from string columns
+    for col in ['Element', 'Symbol', 'Name', 'ChemicalName', 'Type', 'TypeOf']:
+        if col in df:
+            df[col] = df[col].str.strip()
+
+    # Add Type for periodic table without TypeOf column
+    if 'TypeOf' not in df:
+        df['TypeOf'] = 'dcs:ChemicalElement'
```

Comment:
I think we might want to use `dcs:Element` from #111

Reply:
Resolving this comment since [`ChemicalElement`](https://autopush.datacommons.org/browser/ChemicalElement) exists in the KG